basis/common-dao/common-dao-impll/src/main/java/com/softgroup/common/dao/impl/repositories/DeviceRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.softgroup.common.dao.impl.repositories;
+
+import com.softgroup.common.dao.api.entities.DeviceEntity;
+import org.springframework.data.repository.PagingAndSortingRepository;
+
+/**
+ * Created by Виктор on 18.03.2017.
+ */
+public interface DeviceRepository extends PagingAndSortingRepository<DeviceEntity, String>{
```

Comment:
@Mellifaro All our entities will use String ID. Make some base repository to reduce number of generic parameters.